SnapMD.VirtualCare.ApiModels/PatientOnBoardShortDetail.cs

```diff
@@ -53,6 +53,8 @@ public class PatientOnBoardShortDetail
         public PatientOnBoardStatus? Status { get; set; }
         public bool? PreventSendingInvitation { get; set; }
 
+        public bool? AddressValidationExempted { get; set; }
```

Comment:
@jay-lee00 I would prefer if this were a server-side hospital setting rather than something set through the API.